packages/core/src/SimpleTable/index.tsx

```diff
@@ -249,12 +272,19 @@ interface SimpleTableProps extends BaseProps {
    * React element that will go into the top of the table.
    */
   readonly header: React.ReactNode
+  /**
+   * Override default width by fixed width (px) if set.
```

Comment:
As mentioned above, we typically would want a percentage here.

Reply:
Changed to percentage.